src/wazuh_db/wdb_global.c

```diff
@@ -1266,6 +1274,22 @@ wdbc_result wdb_global_unassign_agent_group(wdb_t *wdb, int id, cJSON* j_groups)
     return result;
 }
 
+int wdb_set_default_agent_group(wdb_t *wdb, int id) {
```

Comment:
Use a more clear name since this function does some checks before setting default group.